### PR TITLE
Chore: fix e2e job

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves
The job seems to be failing occasionally due to the yarn cache.

## How this PR fixes it
Hopefully disabling cache should help.

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -15,24 +15,14 @@ jobs:
         with:
           node-version: 16
 
-      - name: Yarn cache
-        uses: actions/cache@v2
-        with:
-          path: '**/node_modules'
-          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
```
[the remark this answers is not in the document]
Comment (katspaugh) [reply .github/workflows/e2e.yml:22]: It was just faster, now you have to wait.

**Comment by mikhailxyz** [comment]
Could you share the failing e2e run? I experienced a yarn cache error before in `safe-react` and may have an idea of how to fix it

**Comment by katspaugh** [comment]
Here, @mikhailxyz: https://github.com/safe-global/web-core/runs/7875778559?check_suite_focus=true

**Comment by mikhailxyz** [comment]
@katspaugh Have you tried any of the steps outlined in the error message?

> The cypress npm package is installed, but the Cypress binary is missing.
> 
> We expected the binary to be installed here: /home/runner/.cache/Cypress/10.4.0/Cypress/Cypress
> 
> Reasons it may be missing:
> 
> - You're caching 'node_modules' but are not caching this path: /home/runner/.cache/Cypress
> - You ran 'npm install' at an earlier build step but did not persist: /home/runner/.cache/Cypress
> 
> Properly caching the binary will fix this error and avoid downloading and unzipping Cypress.
> 
> Alternatively, you can run 'cypress install' to download the binary again.
> 
> https://on.cypress.io/not-installed-ci-error

I'm not sure about caching the cypress binary because I have no idea how to bust the cache when the binary is updated, but the second suggestion about running `cypress install` might be better performance-wise than completely getting rid of the cache and trivial to implement

**Comment by katspaugh** [comment]
@mikhailxyz I don't have capacity to further experiment on this at the moment, but might come back to it when possible.
`cypress install` sounds like a good tip.

**Comment by katspaugh** [comment]
Looks like this is not needed, it works fine in newer PRs.

**Pull request closed without merging**